src/hope/apps/periodic_data_update/celery_tasks.py

```diff
@@ -96,6 +97,7 @@ def merge_pdu_online_edit_task(self: Any, pdu_online_edit_id: int) -> bool:
             service.merge_edit_data()
             pdu_online_edit.status = PDUOnlineEdit.Status.MERGED
             pdu_online_edit.save(update_fields=["status"])
+
         except Exception as e:
             logger.exception("Failed to merge PDU online edit data: %s", e)
```

Comment:
logger.exception() automatically includes traceback so logger.exception("message") not logger.exception(e)